api/v1beta1/types.go

```diff
@@ -635,6 +640,48 @@ type ServiceEndpointSpec struct {
 	Locations []string `json:"locations"`
 }
 
+// PrivateLinkServiceConnection defines the specification for a private link service connection associated with a private endpoint.
+type PrivateLinkServiceConnection struct {
+	// Name specifies the name of the private link service.
+	// +optional
+	Name string `json:"name,omitempty"`
```

Comment:
Yup, it is optional. If not set, the `PrivateLinkServiceConnection` name will be empty.  I've checked on Azure portal and doesn't seem to be displayed anywhere except if you open the json view of the private endpoint resource. I assume the `name` is more used on the filtering side rather than on a critical path